spec/features/staff/users_can_view_forecasts_within_report_spec.rb

```diff
@@ -7,8 +7,27 @@
       authenticate!(user: user)
     end
 
+    def expect_to_see_a_table_of_forecasts_grouped_by_activity(activities)
+      expect(page).to have_content(
+        t("page_content.tab_content.forecasts.per_activity_heading")
+      )
+      activities.each do |activity|
+        within "#activity_#{activity.id}" do
+          expect(page).to have_content(activity.title)
+          expect(page).to have_content(activity.roda_identifier)
+        end
+      end
+    end
+
     scenario "the report contains a _forecasts_ tab" do
-      report = create(:report, :active, organisation: organisation, description: nil)
+      programme = create(:programme_activity)
+
+      report = create(:report, :active, organisation: organisation, fund: programme.parent)
+      project = create(:project_activity, organisation: organisation, parent: programme)
+
+      activities = 2.times.map {
+        create(:third_party_project_activity, organisation: organisation, parent: project)
+      }
```

Comment:
You can use `create_list` for this, i.e: 

```suggestion
      activities = create_list(:third_party_project_activity, 2, organisation: organisation, parent: project)
```

Reply:
I'm not convinced that using `create_list` is an improvements:

Reply:
Ah, gotcha. I made this comment before all the extra bits were added. I think this is fine in that case 👍